pilot/pkg/features/pilot.go

```diff
@@ -640,6 +640,8 @@ var (
 
 	LocalClusterSecretWatcher = env.RegisterBoolVar("LOCAL_CLUSTER_SECRET_WATCHER", false,
 		"If enabled, the cluster secret watcher will watch the namespace of the external cluster instead of config cluster").Get()
+
+	SidecarIgnorePort = env.RegisterBoolVar("SIDECAR_IGNORE_PORT", true, "If enabled, port will not be used in vhost domain matches.").Get()
```

Comment:
SIDECAR_IGNORE_PORT is not clear, 
how about SIDECAR_IGNORE_PORT_IN_HOST_MATCH?

and set to false if we want to back port to release 1.15

Reply:
will rename. I'll set to false on the backport, can we keep true on master? It took me hours to update all the test cases

Reply:
Yes, i think it can be true on master